concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -370,9 +371,24 @@ public ImmutableList<Hash> getAncestors(Hash blockHash, long num) throws BlockNo
                 .orElseThrow(() -> BlockNotFoundException.from(blockHash));
     }
 
+    /**
+     * Get the list of identity providers in the given block.
+     *
+     * @param blockHash Block {@link Hash}.
+     * @return {@link ImmutableList} of {@link IdentityProviderInfo}.
+     */
+    public ImmutableList<IdentityProviderInfo> getIdentityProviders(Hash blockHash) throws Exception {
```

Comment:
Please let the function signature `throws BlockNotFoundException` instead of just `Exception`.

Reply:
done